Error with _ignition tag read ws_ node on 1.5.12

With 1.5.12 of the Ignition Module, updates were performed for the corresponding Jetty 10 migration for Ignition 8.1.25.  Unfortunately, it looks like there is one other issue with the websocket functionality that presents this log message in the gateway upon attempting to connect:

```
[21:29:56]: Map: Error creating servlet node-red-ws 
INFO   | jvm 1    | 2023/02/27 21:29:56 | javax.servlet.ServletException: java.lang.RuntimeException: java.lang.NoSuchMethodException: org.imdc.nodered.servlet.NodeREDWebSocketServlet$NodeREDWebSocketChannel.<init>()
INFO   | jvm 1    | 2023/02/27 21:29:56 | 	at org.eclipse.jetty.websocket.server.JettyWebSocketServlet.init(JettyWebSocketServlet.java:172)
INFO   | jvm 1    | 2023/02/27 21:29:56 | 	at javax.servlet.GenericServlet.init(GenericServlet.java:180)
INFO   | jvm 1    | 2023/02/27 21:29:56 | 	at com.inductiveautomation.ignition.gateway.bootstrap.MapServlet.service(MapServlet.java:75)
INFO   | jvm 1    | 2023/02/27 21:29:56 | 	at org.eclipse.jetty.servlet.ServletHolder$NotAsync.service(ServletHolder.java:1410)
INFO   | jvm 1    | 2023/02/27 21:29:56 | 	at org.eclipse.jetty.servlet.ServletHolder.handle(ServletHolder.java:764)
INFO   | jvm 1    | 2023/02/27 21:29:56 | 	at org.eclipse.jetty.servlet.ServletHandler$ChainEnd.doFilter(ServletHandler.java:1665)
INFO   | jvm 1    | 2023/02/27 21:29:56 | 	at org.eclipse.jetty.servlet.ServletHandler.doHandle(ServletHandler.java:527)
INFO   | jvm 1    | 2023/02/27 21:29:56 | 	at org.eclipse.jetty.server.handler.ScopedHandler.handle(ScopedHandler.java:131)
INFO   | jvm 1    | 2023/02/27 21:29:56 | 	at org.eclipse.jetty.security.SecurityHandler.handle(SecurityHandler.java:578)
INFO   | jvm 1    | 2023/02/27 21:29:56 | 	at org.eclipse.jetty.server.handler.HandlerWrapper.handle(HandlerWrapper.java:122)
INFO   | jvm 1    | 2023/02/27 21:29:56 | 	at org.eclipse.jetty.server.handler.ScopedHandler.nextHandle(ScopedHandler.java:223)
INFO   | jvm 1    | 2023/02/27 21:29:56 | 	at org.eclipse.jetty.server.session.SessionHandler.doHandle(SessionHandler.java:1570)
INFO   | jvm 1    | 2023/02/27 21:29:56 | 	at org.eclipse.jetty.server.handler.ScopedHandler.nextHandle(ScopedHandler.java:221)
INFO   | jvm 1    | 2023/02/27 21:29:56 | 	at org.eclipse.jetty.server.handler.ContextHandler.doHandle(ContextHandler.java:1383)
INFO   | jvm 1    | 2023/02/27 21:29:56 | 	at org.eclipse.jetty.server.handler.ScopedHandler.nextScope(ScopedHandler.java:176)
INFO   | jvm 1    | 2023/02/27 21:29:56 | 	at org.eclipse.jetty.servlet.ServletHandler.doScope(ServletHandler.java:484)
INFO   | jvm 1    | 2023/02/27 21:29:56 | 	at org.eclipse.jetty.server.session.SessionHandler.doScope(SessionHandler.java:1543)
INFO   | jvm 1    | 2023/02/27 21:29:56 | 	at org.eclipse.jetty.server.handler.ScopedHandler.nextScope(ScopedHandler.java:174)
INFO   | jvm 1    | 2023/02/27 21:29:56 | 	at org.eclipse.jetty.server.handler.ContextHandler.doScope(ContextHandler.java:1305)
INFO   | jvm 1    | 2023/02/27 21:29:56 | 	at org.eclipse.jetty.server.handler.ScopedHandler.handle(ScopedHandler.java:129)
INFO   | jvm 1    | 2023/02/27 21:29:56 | 	at org.eclipse.jetty.server.handler.HandlerWrapper.handle(HandlerWrapper.java:122)
INFO   | jvm 1    | 2023/02/27 21:29:56 | 	at com.inductiveautomation.catapult.handlers.RemoteHostNameLookupHandler.handle(RemoteHostNameLookupHandler.java:121)
INFO   | jvm 1    | 2023/02/27 21:29:56 | 	at org.eclipse.jetty.server.handler.HandlerWrapper.handle(HandlerWrapper.java:122)
INFO   | jvm 1    | 2023/02/27 21:29:56 | 	at org.eclipse.jetty.rewrite.handler.RewriteHandler.handle(RewriteHandler.java:301)
INFO   | jvm 1    | 2023/02/27 21:29:56 | 	at org.eclipse.jetty.server.handler.HandlerList.handle(HandlerList.java:51)
INFO   | jvm 1    | 2023/02/27 21:29:56 | 	at org.eclipse.jetty.server.handler.HandlerCollection.handle(HandlerCollection.java:141)
INFO   | jvm 1    | 2023/02/27 21:29:56 | 	at org.eclipse.jetty.server.handler.HandlerWrapper.handle(HandlerWrapper.java:122)
INFO   | jvm 1    | 2023/02/27 21:29:56 | 	at org.eclipse.jetty.server.Server.handle(Server.java:563)
INFO   | jvm 1    | 2023/02/27 21:29:56 | 	at org.eclipse.jetty.server.HttpChannel.lambda$handle$0(HttpChannel.java:505)
INFO   | jvm 1    | 2023/02/27 21:29:56 | 	at org.eclipse.jetty.server.HttpChannel.dispatch(HttpChannel.java:762)
INFO   | jvm 1    | 2023/02/27 21:29:56 | 	at org.eclipse.jetty.server.HttpChannel.handle(HttpChannel.java:497)
INFO   | jvm 1    | 2023/02/27 21:29:56 | 	at org.eclipse.jetty.server.HttpConnection.onFillable(HttpConnection.java:282)
INFO   | jvm 1    | 2023/02/27 21:29:56 | 	at org.eclipse.jetty.io.AbstractConnection$ReadCallback.succeeded(AbstractConnection.java:314)
INFO   | jvm 1    | 2023/02/27 21:29:56 | 	at org.eclipse.jetty.io.FillInterest.fillable(FillInterest.java:100)
INFO   | jvm 1    | 2023/02/27 21:29:56 | 	at org.eclipse.jetty.io.SelectableChannelEndPoint$1.run(SelectableChannelEndPoint.java:53)
INFO   | jvm 1    | 2023/02/27 21:29:56 | 	at org.eclipse.jetty.util.thread.strategy.AdaptiveExecutionStrategy.runTask(AdaptiveExecutionStrategy.java:416)
INFO   | jvm 1    | 2023/02/27 21:29:56 | 	at org.eclipse.jetty.util.thread.strategy.AdaptiveExecutionStrategy.consumeTask(AdaptiveExecutionStrategy.java:385)
INFO   | jvm 1    | 2023/02/27 21:29:56 | 	at org.eclipse.jetty.util.thread.strategy.AdaptiveExecutionStrategy.tryProduce(AdaptiveExecutionStrategy.java:272)
INFO   | jvm 1    | 2023/02/27 21:29:56 | 	at org.eclipse.jetty.util.thread.strategy.AdaptiveExecutionStrategy.lambda$new$0(AdaptiveExecutionStrategy.java:140)
INFO   | jvm 1    | 2023/02/27 21:29:56 | 	at org.eclipse.jetty.util.thread.ReservedThreadExecutor$ReservedThread.run(ReservedThreadExecutor.java:411)
INFO   | jvm 1    | 2023/02/27 21:29:56 | 	at org.eclipse.jetty.util.thread.QueuedThreadPool.runJob(QueuedThreadPool.java:934)
INFO   | jvm 1    | 2023/02/27 21:29:56 | 	at org.eclipse.jetty.util.thread.QueuedThreadPool$Runner.run(QueuedThreadPool.java:1078)
INFO   | jvm 1    | 2023/02/27 21:29:56 | 	at java.base/java.lang.Thread.run(Unknown Source)
INFO   | jvm 1    | 2023/02/27 21:29:56 | Caused by: java.lang.RuntimeException: java.lang.NoSuchMethodException: org.imdc.nodered.servlet.NodeREDWebSocketServlet$NodeREDWebSocketChannel.<init>()
INFO   | jvm 1    | 2023/02/27 21:29:56 | 	at org.eclipse.jetty.websocket.server.JettyWebSocketServlet$CustomizedWebSocketServletFactory.register(JettyWebSocketServlet.java:218)
INFO   | jvm 1    | 2023/02/27 21:29:56 | 	at org.imdc.nodered.servlet.NodeREDWebSocketServlet.configure(NodeREDWebSocketServlet.java:37)
INFO   | jvm 1    | 2023/02/27 21:29:56 | 	at org.eclipse.jetty.websocket.server.JettyWebSocketServlet.init(JettyWebSocketServlet.java:168)
INFO   | jvm 1    | 2023/02/27 21:29:56 | 	... 42 common frames omitted
INFO   | jvm 1    | 2023/02/27 21:29:56 | Caused by: java.lang.NoSuchMethodException: org.imdc.nodered.servlet.NodeREDWebSocketServlet$NodeREDWebSocketChannel.<init>()
INFO   | jvm 1    | 2023/02/27 21:29:56 | 	at java.base/java.lang.Class.getConstructor0(Unknown Source)
INFO   | jvm 1    | 2023/02/27 21:29:56 | 	at java.base/java.lang.Class.getDeclaredConstructor(Unknown Source)
INFO   | jvm 1    | 2023/02/27 21:29:56 | 	at org.eclipse.jetty.websocket.server.JettyWebSocketServlet$CustomizedWebSocketServletFactory.register(JettyWebSocketServlet.java:214)
INFO   | jvm 1    | 2023/02/27 21:29:56 | 	... 44 common frames omitted
```

--- a/IgnitionNodeRED-build/pom.xml
+++ b/IgnitionNodeRED-build/pom.xml
@@ -47,7 +47,7 @@
                     <moduleId>org.imdc.nodered.IgnitionNodeRED</moduleId>
                     <moduleName>${module-name}</moduleName>
                     <moduleDescription>${module-description}</moduleDescription>
-                    <moduleVersion>1.5.12.${timestamp}</moduleVersion>
+                    <moduleVersion>1.5.13.${timestamp}</moduleVersion>
                     <requiredIgnitionVersion>${ignition-platform-version}</requiredIgnitionVersion>
                     <requiredFrameworkVersion>8</requiredFrameworkVersion>
                     <licenseFile>license.html</licenseFile>

--- a/IgnitionNodeRED-gateway/src/main/java/org/imdc/nodered/servlet/NodeREDWebSocketServlet.java
+++ b/IgnitionNodeRED-gateway/src/main/java/org/imdc/nodered/servlet/NodeREDWebSocketServlet.java
@@ -34,7 +34,6 @@ public class NodeREDWebSocketServlet extends JettyWebSocketServlet {
     @Override
     public void configure(JettyWebSocketServletFactory factory) {
         factory.setMaxTextMessageSize(2 * MB);
-        factory.register(NodeREDWebSocketChannel.class);
         factory.setCreator(new NodeREDWebSocketCreator(getContext()));
     }
 

--- a/node-red-contrib-ignition/CHANGELOG.md
+++ b/node-red-contrib-ignition/CHANGELOG.md
@@ -37,3 +37,9 @@ August 15, 2021
 August 20, 2021
 
  * Added dependencies for ws, https-proxy-agent, and url
+
+### 1.5.13
+
+February 27, 2023
+
+ * Version bump only for alignment with Ignition module and support for 8.1.25+

--- a/node-red-contrib-ignition/package.json
+++ b/node-red-contrib-ignition/package.json
@@ -1,6 +1,6 @@
 {
   "name": "node-red-contrib-ignition-nodes",
-  "version": "1.5.11",
+  "version": "1.5.13",
   "description": "Adds support for reading, writing, and browsing tags in Ignition",
   "dependencies": {
 	  "ws": "^7.5.3",

